shuttle/templates/schedules/load_logs.html

```diff
@@ -0,0 +1,100 @@
+{% block body_content %}
+    {% if shuttle_logs %}
+        <h4>Driver Shuttle Logs</h4>
+        <div class="card">
+        <div class="card-header" id="heading-tw">
+            {{ date }}
+        </div>
+        <div class="card-body log-body">
+        <table class="table table-hover">
+        <thead>
+        <tr>
+            <th scope="col">Driver</th>
+            <th scope="col">Location</th>
+            <th scope="col">Check In</th>
+            <th scope="col">Check Out</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            {% for i in range(shuttle_logs|length) %}
+                <td class="driver">
+                    {% if shuttle_logs[i]['name'] != None %}
+                        {{ shuttle_logs[i]['name'] }}
+                    {% endif %}
+                </td>
+                 <td class="location">
+                    {% if shuttle_logs[i]['location'] != None %}
+                        {{ shuttle_logs[i]['location'] }}
+                    {% endif %}
+                </td>
+                <td class="check-in">
+                    {% if shuttle_logs[i]['arrival_time'] != None %}
+                        {{ shuttle_logs[i]['arrival_time'] }}
+                    {% endif %}
+                </td>
+                <td class="check-out">
+                    {% if shuttle_logs[i]['departure_time'] != None %}
+                        {{ shuttle_logs[i]['departure_time'] }}
+                    {% endif %}
+                </td>
+                </tr>
+            {% endfor %}
+    {% endif %}
+</tbody>
+</table>
+</div>
+</div>
+
+    {% if shuttle_logs and break_logs %}
+        <hr>
+    {% endif %}
+
+    {% if break_logs %}
+        <h4>Driver Break Logs</h4>
+        <div class="card">
+        <div class="card-header" id="heading-tw">
+            {{ date }}
+        </div>
+        <div class="card-body log-body">
+        <table class="table table-hover">
+        <thead>
+        <tr>
+            <th scope="col">Driver</th>
+            <th scope="col">Break Time In</th>
+            <th scope="col">Break Time Out</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            {% for i in range(break_logs|length) %}
+                <td class="driver">
+                    {% if break_logs[i]['name'] != None %}
+                        {{ break_logs[i]['name'] }}
+                    {% endif %}
+                </td>
+                <td class="check-in">
+                    {% if break_logs[i]['arrival_time'] != None %}
+                        {{ break_logs[i]['arrival_time'] }}
+                    {% endif %}
+                </td>
+                <td class="check-out">
+                    {% if break_logs[i]['departure_time'] != None %}
+                        {{ break_logs[i]['departure_time'] }}
+                    {% endif %}
+                </td>
+                </tr>
+            {% endfor %}
+    {% endif %}
+</tbody>
+</table>
+</div>
+</div>
```

Comment:
These look like they should be inside the if statement not outside it.

Reply:
Ope, you right